README.md

```diff
@@ -187,6 +187,28 @@ randomString
 - npm install 
 - npm run test
 
+## Command Line Usage
+```js
+$ npm install randomized-string
+
+$ yarn add randomized-string
```

Comment:
Command-line works if the package is installed globally.so if you make it `npm install randomized-string -g` or  `yarn add randomized-string -g`  it will work as global dependency